reviews/EJOR/02/ResponsetoReviewers.tex

```diff
@@ -0,0 +1,102 @@
+        DOI:10.1080/00207543.2021.1992529}
+    \end{itemize}
+
+    These two papers are now cited in the manuscript at the literature review
+    section.
+
+    \line(1, 0){0.92\textwidth}
+    \begin{itemize}
+        \item Reviewer 2: I gave a look into the new version and the comments
+        made by the authors following the reviews they received.
+        My criticism towards the paper as stated in the first round stays.
+        It is basically the question of what are the standards of EJOR.
+        If a paper which states a interesting model and gives details on
+        numerical procedures for solving it but without any theorems and
+        insights is good enough, then it should be accepted as the paper is
+        doing a good job of what it promises.
+        Otherwise, they should look for a more applied journal where what they
+        do is sufficient.
+    \end{itemize}
+
+    Here's a list of reasons of why I think this paper should be considered for
+    publication in EJOR.
+    Firstly, reviewer 2 states that the paper does not provide any insights.
+    We would like to point out that this paper does provide theoretical results
+    and insights.
+    Section 3 provides detailed theoretical expressions of the queueing model's
+    performance measures.
+    Expressions of the waiting time, blocking time and proportion of
+    individuals within target have been derived and are presented in the paper.
+    Also, section 4.2 contains a list of applied scenarios where insights 
+    are presented.
+    A busy model is examined and insights on ways to increase the relative
+    efficiency of the model are presented.
+
+    Secondly here is a list of papers already published in EJOR that introduce
+    a new model and give details on numerical procedures for solving it but
+    without any new theorems:
+    \begin{itemize}
+        \item A queueing model for general group screening policies and dynamic
+        item arrivals \\
+        https://doi.org/10.1016/j.ejor.2010.05.042
+        \item Selfish routing in public services: \\
+        https://doi.org/10.1016/j.ejor.2013.04.003
+        \item The effect of ambulance relocations on the performance of
+        ambulance service providers \\
+        https://doi.org/10.1016/j.ejor.2015.12.022
+        \item Robust and stochastic formulations for ambulance deployment and
+        dispatch \\
+        https://doi.org/10.1016/j.ejor.2019.05.011
+        \item A Markovian queueing model for ambulance offload delays \\
+        https://doi.org/10.1016/j.ejor.2012.11.030
+        \item Models and algorithms for an integrated vessel scheduling and tug
+        assignment problem within a canal harbor \\
+        https://doi.org/10.1016/j.ejor.2021.10.037
+        \item The application of operational research to European air traffic
+        flow management - understanding the context \\
+        https://doi.org/10.1016/S0377-2217(99)00084-3
```

Comment:
Could you add the title/authors etc here. Just to make it less effort for the reviewer.